<?xml version="1.0"?>
<launch>
  <param name="map_name"  value="map7"/>
  <param name="inicio_x"  value="3"/>
  <param name="inicio_y"  value="9"/>
  
  <!-- Servicios -->
  <node pkg="planificacion_pkg" name="lector_mapa_csv" type="lector_mapa_csv"/>
  
  <!-- Planificador -->
  <node pkg="planificacion_pkg" name="planificador_aestrella" type="planificador_aestrella" output="screen" launch-prefix="xterm -e"/>

</launch>
